providers/configure.rb

```diff
@@ -108,7 +108,14 @@ def configure
         shell current['shell']
         system current['systemuser']
         uid current['uid'] unless current['uid'].nil?
-        not_if { node['etc']['passwd'][current['user']] }
+
+        not_if do
+          begin
+            Etc.getpwnam current['user']
```

Comment:
Probably want to add a `true` on the next line out put the in `!!()` so that you don't get a warning about the `not_if` returning something other than a boolean (`not_if block for XX returned XX, did you mean to run a command?`)